src/Controllers/ItemController/item.ts

```diff
@@ -22,3 +26,24 @@ export const createItem = (req: Request, res: Response) => {
             res.status(400).json(error);
         });
 };
+
+
+
+export const getTotalItems = async (req: Request, res: Response) => {
+   const items = await getAllItems(req, res);
```

Comment:
This needs to be paginated

Reply:
The pagination was added as a middleware in the routes folder..

Reply:
Ok